CHANGELOG.md

```diff
@@ -2,8 +2,18 @@
 
 ## [unreleased]
 
+### Protocol Changes
+**No Changes**
+
+### Non-protocol Changes
+**No Changes**
+
+## 2.30
+
 ### Protocol Changes
 * Sets `chunk_validator_only_kickout_threshold` to 70. Uses this kickout threshold as a cutoff threshold for contribution of endorsement ratio in rewards calculation: if endorsement ratio is above 70%, the contribution of endorsement ratio in average uptime calculation is 100%, otherwise it is 0%. Endorsements received are now included in `BlockHeader` to improve kickout and reward calculation for chunk validators. 
+* Changes the structure of ChunkEndorsements
```

Comment:
I think this should come under non-protocol changes section. We are changing the network message tho, but that isn't a part of protocol I believe?

Reply:
yeah i just added it without looking too deeply because I saw it's included in the protocol features but you're right, def not strictly a protocol change. Actually let's just delete this part. I think it's prob not super relevant for most node operators.